pegasus/sites.v3/code.org/public/helloworld.md.partial

```diff
@@ -37,3 +37,5 @@ In Computer Science 101, the first program many students create is a simple one
 {{ featured_project, title: "Animals", author: "B", age: "18+", image: "images/csc/helloworld/cschelloworld_animals.gif", alt_text: "A student project featuring an animated gif of an underwater scene with dolphin, fish, and jellyfish characters in front of a shipwreck", url: "https://studio.code.org/projects/spritelab/rYH8D8eAvWOjuiOpWbHzN4HAtis4ykKTIjGcNPP9zD4" }}
 {{ featured_project, title: "Retro", author: "W", age: "13+", image: "images/csc/helloworld/cschelloworld_retro.gif", alt_text: "A student project featuring an animated gif of a game of tag with two alien characters on a distant planet", url: "https://studio.code.org/projects/spritelab/rYH8D8eAvWOjuiOpWbHzN7yo2E1S0q87VqlzaBz7oqo" }}
 {{ clearfix }}
+
+{{ helloworld }}
```

Comment:
I think until this file gets re-translated (which can take a really long time, unless we deliberately flush the translations out) then a non-en user will see the old version of this file, albeit with new CSS around it.  Will that look alright?

cc @code-dot-org/i18n

Reply:
@code-dot-org/i18n ping

Reply:
I think the easiest path forward would be to find a way to update these without touching any of the `.md.partial` files...

Reply:
I'm not sure that's possible(

Reply:
I see 2 options here, but only one is workable( :
1. Edit one of components/partials used in this file before, however it will affect all of the pages that render that component/partial
2. (the one that's implemented right now) create new component/partial with all the specific html for that page

Reply:
And, while I was writing previous comment I've thought of making some global js logic that will inject html/style directly into the rendered page. However, I'm not sure if it's worth it

Reply:
I've also asked @kelbyhawn , but she had no ideas as well at that time.

However, the issue is indeed there. This way styles will affect only eng version of the site. If you know who we/I can ask about how to fix it, I'll be much obliged)

![Screenshot from 2023-02-09 23-06-16](https://user-images.githubusercontent.com/22244040/217939757-80f1947a-f87b-4244-8337-3627f8aba008.png)
![Screenshot from 2023-02-09 23-06-39](https://user-images.githubusercontent.com/22244040/217939814-a41ee4bc-1d7d-4599-ad7b-ca2cd35561f8.png)

Reply:
Okay, it looks like the best solution will be to add this new entry to the file, since (from offline discussions) the page will look alright (but older) without it, and once each language gets an updated "translation" of the page, it will get the newer look.